src/utils/browserUtil.ts

```diff
@@ -0,0 +1,9 @@
+import * as openpackage from 'open';
```

Comment:
With this approach, we're always loading the module even if we don't need it. Since you won't need opening the browser on every CLI execution, let's change it into lazy loading like we did originally so that we only load the package when it's actually needed. This will help us keep the CLI fast.